go/kbfs/libkbfs/prefetcher.go

```diff
@@ -106,6 +107,9 @@ type blockPrefetcher struct {
 	doneCh chan struct{}
 	// map to store prefetch metadata
 	prefetches map[kbfsblock.ID]*prefetch
+	// map to channels for cancelling queued prefetches
+	queuedPrefetchHandles     map[kbfsblock.ID]chan<- struct{}
+	queuedPrefetchHandlesLock *sync.Mutex
```

Comment:
Should not be a pointer, and thus shouldn't need to be initialized in the constructor below.